test/WebJobs.Script.Tests.E2E/GeneralEndToEndTests.cs

```diff
@@ -107,6 +110,85 @@ public GeneralEndToEndTests(FunctionAppFixture fixture)
             }
         }
 
+        [Fact]
+        [TestTrace]
```

Comment:
That is a custom attribute that logs all test runs (duration) using the telemetry client, which pushes the data to AI.